conf/application.conf

```diff
@@ -1,49 +1,124 @@
 # This is the main configuration file for the application.
 # ~~~~~
 
-# Secret key
+
+# PRODUCTS
 # ~~~~~
-# The secret key is used to secure cryptographics functions.
-# If you deploy your application to several instances be sure to use the same key!
-play.crypto.secret="changeme"
-play.crypto.secret=${?APPLICATION_SECRET}
 
-# Injection configuration
+# Product attribute list that is displayed for a specific product variant (this does not allow to select different
+# product variants, please check soft/hard selectable attributes for that purpose). Enter a list of the attribute keys.
+productData.displayedAttributes = ["colorFreeDefinition", "size", "designer"]
+productData.displayedAttributes = ${?DISPLAYED_ATTRIBUTES}
+
+# Product attribute list that allows to select a clearly different variant of your product, which requires to load the
+# new product information associated with it (e.g. color). Enter a list of the attribute keys.
+productData.hardSelectableAttributes = ["color"]
+productData.hardSelectableAttributes = ${?HARD_SELECTABLE_ATTRIBUTES}
+
+# Product attribute list that allows to select a specific variant, but does not require to change the displayed product
```

Comment:
I don't like the term change here since it could sound like updating a product, how about `reload the displayed product for the selected variant`?